src/com/company/reverse/Reverse.java

```diff
@@ -0,0 +1,21 @@
+package com.company.reverse;
+
+public class Reverse {
+    public static void main(String[] args) {
+
+
+        System.out.println(reverse("ad12fgiu57"));
+    }
+
+    private static String reverse(String s) {
```

Comment:
Please provide more meaningful name for parameter ‘s’

Reply:
Done